edit-post/components/block-settings-menu/plugin-block-settings-menu-item.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import { difference } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { IconButton } from '@wordpress/components';
+import { compose } from '@wordpress/element';
```

Comment:
Similarly, `wp.element.compose` was deprecated in v3.3 and is slated to be removed in the next release. See #7948